### ISSUE TITLE
AttributeError: module 'numpy.core.umath' has no attribute 'divmod'

### ISSUE THREAD

**Issue by michal-pikusa** [HazyResearch/snorkel#1052]
While running CDR_Tutorial_2 I came across an error, while executing:

```
from snorkel.annotations import LabelAnnotator
labeler = LabelAnnotator(lfs=LFs)
```

Here's the whole traceback:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-10-7be22b187076> in <module>
----> 1 from snorkel.annotations import LabelAnnotator
      2 labeler = LabelAnnotator(lfs=LFs)

~\Anaconda3\envs\snorkel\lib\site-packages\snorkel\annotations.py in <module>
      7 
      8 import numpy as np
----> 9 from pandas import DataFrame, Series
     10 import scipy.sparse as sparse
     11 from sqlalchemy.sql import bindparam, select

~\Anaconda3\envs\snorkel\lib\site-packages\pandas\__init__.py in <module>
     40 import pandas.core.config_init
     41 
---> 42 from pandas.core.api import *
     43 from pandas.core.sparse.api import *
     44 from pandas.tseries.api import *

~\Anaconda3\envs\snorkel\lib\site-packages\pandas\core\api.py in <module>
      5 import numpy as np
      6 
----> 7 from pandas.core.arrays.integer import (
      8     Int8Dtype,
      9     Int16Dtype,

~\Anaconda3\envs\snorkel\lib\site-packages\pandas\core\arrays\__init__.py in <module>
     11     IntegerArray, integer_array)
     12 from .sparse import SparseArray  # noqa
---> 13 from .numpy_ import PandasArray, PandasDtype  # noqa

~\Anaconda3\envs\snorkel\lib\site-packages\pandas\core\arrays\numpy_.py in <module>
     90 # So, we fall back to "object", to avoid writing a metaclass.
     91 try:
---> 92     from numpy.lib.mixins import NDArrayOperatorsMixin
     93 except ImportError:
     94     NDArrayOperatorsMixin = object

~\Anaconda3\envs\snorkel\lib\site-packages\numpy\lib\mixins.py in <module>
     61 
     62 
---> 63 class NDArrayOperatorsMixin(object):
     64     """Mixin defining all operator special methods using __array_ufunc__.
     65 

~\Anaconda3\envs\snorkel\lib\site-packages\numpy\lib\mixins.py in NDArrayOperatorsMixin()
    164         um.floor_divide, 'floordiv')
    165     __mod__, __rmod__, __imod__ = _numeric_methods(um.remainder, 'mod')
--> 166     __divmod__ = _binary_method(um.divmod, 'divmod')
    167     __rdivmod__ = _reflected_binary_method(um.divmod, 'divmod')
    168     # __idivmod__ does not exist

AttributeError: module 'numpy.core.umath' has no attribute 'divmod'
```

Is there a way to fix this error? Thanks.

**Comment by bhancock8** [comment]
It looks to me like you may be using an out of date version of numpy. Try upgrading your numpy package (e.g. `conda update numpy` or `pip install -U numpy`). If that doesn't work, you may have multiple versions of numpy on your path---make sure that you're using the one you intend to be using. Let us know if this solves your issue.

**Comment by michal-pikusa** [comment]
Thanks for a quick reply. Upgrading numpy helped, and I was able to finish the rest of the tutorial.

I was using a conda environment created with your environment.yml, and when I looked at the file, it says 
```- numpy>=1.11```
but I was using 1.15, and still got the error. Now I'm using 1.16.1 and it works, so you might look into updating the environment.yml on the repository. Thanks for your help!
